internal/xpkg/fetch.go

```diff
@@ -143,6 +157,26 @@ func (i *K8sFetcher) Fetch(ctx context.Context, ref name.Reference, secrets ...s
 	if err != nil {
 		return nil, err
 	}
+	if len(i.validationPem) != 0 {
```

Comment:
Do you think that we'll want to use one pub key and apply to all packages? I could see a use-case for verifying some packages are signed but not others. Does cosign support loading multiple signature verifiers? One option for keeping this flexible as an alpha feature would be to support an annotation on `Provider` and `Configuration` to indicate whether they should have their signature verified.

Reply:
I like that but will need some direction on that. Would that need to get replicated to the revision's of them as well? If I understand the current architecture, the package types create revision types that are what are used by the reconciler to know which packages to pull?